esp/services/ws_store/espstorelib/daliKVStore.cpp

```diff
@@ -57,6 +63,13 @@ bool CDALIKVStore::createStore(const char * apptype, const char * storename, con
     if (maxvalsize != 0)
         apptree->setPropInt(DALI_KVSTORE_MAXVALSIZE_ATT, maxvalsize);
 
+    Owned<IPropertyTree> root = conn->getRoot();
+    if (root->hasProp(xpath.str()))
+    {
+        LOG(MCuserInfo,"DALI Keystore createStore(): '%s' entry already exists", storename);
+        return false;
+    }
+
     root->addPropTree("Store", LINK(apptree));
```

Comment:
trivial/no need to change: trivially more efficient to do apptree.getClear() here.